CODEOWNERS

```diff
@@ -109,6 +109,7 @@
 # Internal Snuba topics
 /topics/snuba-queries.yaml                                     @getsentry/owners-snuba
 /topics/event-replacements.yaml                                @getsentry/owners-snuba
+/topics/snuba-lw-deletions.yaml                                @getsentry/owners-snuba
```

Comment:
are these deletions supposed to be for a specific storage? i'd suggest putting that in the name of the topic to make it clear to everyone + if you may have multiple deletions topics on different datasets in the future

Reply:
@lynnagara I was planning on using one topic for all storages and have the consumer group determine which messages to process (based on the storage, which is why I was asking about the kafka headers in slack)

Reply:
i see, please use separate topic instead. we don't have shared topics across pipelines, and i don't think we should start here. topics are cheap and easy to create. it will be more efficient for your consumers anyway to not have to download and manually handle all the messages intended for other pipelines.

Reply:
you want to have a single consumer subscribe to all the topics and do every type of deletions?

we don't really do this today except in the post process forwarder, but it's fine to change, and we probably should anyway to align to the kafka consumer apis which all support this